src/SFA.DAS.LevyTransferMatching.Web/Helpers/UnorderedListTagHelper.cs

```diff
@@ -1,26 +1,31 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Text;
 using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.AspNetCore.Mvc.ViewFeatures;
 using Microsoft.AspNetCore.Razor.TagHelpers;
+using SFA.DAS.LevyTransferMatching.Infrastructure.Tags;
 using SFA.DAS.LevyTransferMatching.Web.Extensions;
 
 namespace SFA.DAS.LevyTransferMatching.Web.Helpers
 {
-    public class FlagsUnorderedListTagHelper : TagHelper
+    public class UnorderedListTagHelper : TagHelper
     {
         [HtmlAttributeName("asp-for")]
         public ModelExpression Property { get; set; }
 
+        [HtmlAttributeName("source")]
+        public List<Tag> Source { get; set; }
+
         [HtmlAttributeName("class")]
         public string Class { get; set; }
 
         [HtmlAttributeName("item-class")]
         public string ItemClass { get; set; }
 
-        [ViewContext]
-        [HtmlAttributeNotBound]
-        public ViewContext ViewContext { get; set; }
+        //[ViewContext]
+        //[HtmlAttributeNotBound]
+        //public ViewContext ViewContext { get; set; }
```

Comment:
If this isn't needed can we delete it rather than commenting out?

Reply:
Yes, my bad